src/refactorings/change-signature/change-signature.modules.test.ts

```diff
@@ -0,0 +1,634 @@
+import { AbsolutePath } from "../../editor/path";
+import { testEach } from "../../tests-helpers";
+import { InMemoryEditor } from "../../editor/adapters/in-memory-editor";
+import { changeSignature } from "./change-signature";
+import { Code, SelectedPosition } from "../../editor/editor";
+
+type TestSample = {
+  code: Code;
+  path: AbsolutePath;
+};
+
+describe("Modules", () => {
```

Comment:
👍 For splitting the tests in a meaningful way

I would simply rename this to be more explicit in tests report:

```suggestion
describe("Change Signature: Modules", () => {
```